app/presenters/organization_presenter.rb

```diff
@@ -0,0 +1,9 @@
+class OrganizationPresenter < Burgundy::Item
+  def members
+    UserPresenter.wrap item.members.sort_by(&:last_name)
+  end
+
+  def developments
+    item.developments.uniq
+  end
+end
```

Comment:
A lot of files don't have newlines at the end. I didn't know why this was a thing [until reading this just now](https://robots.thoughtbot.com/no-newline-at-end-of-file), which I recommend also reading (and there's instructions on how to ensure this in Sublime).